spec/money_column_spec.rb

```diff
@@ -139,9 +139,8 @@ class MoneyClassInheritance2 < MoneyWithCustomAccessors
     expect(record.currency.to_s).to eq('EUR')
   end
 
-  it 'does overwrite a currency if changed but will show a deprecation notice' do
+  it 'does overwrite a currency' do
     expect(record.currency.to_s).to eq('EUR')
-    expect(Money).to receive(:deprecate).once
```

Comment:
Would be good for completeness sake to keep this old test around with the new test.

Reply:
not sure what you mean, the old test doesn't pass anymore since we don't want to show the deprecation

Reply:
Sorry, meant the old test could be kept around if we added the `currency_read_only` option to test the deprecation.

Reply:
ah, sorry makes sense. We are already covered by this one: https://github.com/Shopify/money/pull/171/files#diff-1a873d5a205667d2b652832f32bdc7c6R220